src/Libraries/Microsoft.Extensions.Compliance.Abstractions/README.md

```diff
@@ -1 +1,65 @@
-README
+# Microsoft.Extensions.Compliance.Abstractions
+
+This provides types necessary to extend the redaction feature.
```

Comment:
This doesn't extend, this introduces redaction abstractions. Perhaps:

This package introduces data classification and data redaction features.